code401challenges/src/test/java/code401challenges/BinarySearchTest.java

```diff
@@ -0,0 +1,17 @@
+package code401challenges;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class BinarySearchTest {
+    @Test public void testBinarySearch() {
+        int[] inputArray = {4, 8, 26, 15, 43, 18, 10, 12};
+        int inputNumber = 15;
+        int output  = 3;
+        int actual = BinarySearch.binarySearch(inputArray, inputNumber);
+
+        assertEquals("Expected 15", output, actual);
```

Comment:
A good single test to start with! Note that the testing requirements say you should, at minimum, test for a happy path and an expected failure (like a missing number).